fluentd/lib/in_tail_metric/write_watcher.rb

```diff
@@ -17,9 +17,8 @@
 require 'fluent/plugin/input'
 
 module Fluent::Plugin
-  class TailInput < Fluent::Plugin::Input
```

Comment:
We can drop this watcher since it' doesn't work correctly. The Go process will generate these metrics.